### DISCUSSION TITLE
元数据为什么会变成readonly

### DISCUSSION THREAD

**Discussion by kenny286** [cubefs/cubefs#3454] [General]
![image](https://github.com/user-attachments/assets/a673c27c-fe8c-490a-8cd5-0f2b7d4b7ede)

![image](https://github.com/user-attachments/assets/3a7d7e16-0f72-4898-b193-bea7ef748114)

**Comment by leonrayang** [comment]
As mentioned in the users' WeChat group, the memory allocated for the metanode should not be too low. Typically, the limit for the metanode size should be set to 70% of the node's total memory, while ensuring that at least 1GB remains available for the metanode to prevent excessive memory usage and exhaustion.

**Comment by leonrayang** [comment]
The last partition is always writable for system availability.
